recipe-server/normandy/base/utils.py

```diff
@@ -82,5 +82,7 @@ def sri_hash(data):
     :param data:
         Bytes-like object containing the data to hash.
     """
-    data_hash = b64encode(sha384(data).digest())
-    return 'sha384-' + data_hash.decode()
+    data_hash = b64encode(sha384(data).digest()).decode()
+    if url_safe:
+        data_hash = data_hash.replace('+', '-').replace('/', '_')
```

Comment:
I'm pretty sure that SRI doesn't support url safe base64. At least, [the spec](https://www.w3.org/TR/SRI/) doesn't talk about it.